bot/exts/backend/error_handler.py

```diff
@@ -164,6 +168,10 @@ async def try_get_tag(self, ctx: Context) -> None:
         the context to prevent infinite recursion in the case of a CommandNotFound exception.
         """
         tags_get_command = self.bot.get_command("tags get")
+        if not tags_get_command:
+            log.debug("Not attempting to parse message as a tag as could not find `tags get` command.")
+            return False
```

Comment:
This function returns `None` on failure and success.
```suggestion
            return
```